File: src/main/java/com/musiqueando/domain/Sonido.java
package com.musiqueando.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Document(collection = "sonidos")
public class Sonido {
    @Id
    private String id;
    private String nombre;
    private String categoria;
    private double duracion;
    private String path;
}

File: app/src/main/java/com/example/hospitalcompose/domain/modelo/Sonido.kt
package com.example.hospitalcompose.domain.modelo

import kotlinx.serialization.Serializable
import kotlinx.serialization.SerialName


@Serializable
data class Sonido(
    @SerialName("_id")
    val id: String,
    val nombre: String,
    val categoria: String,
    val duracion: Double,
    val path: String
)
